src/__tests__/02b-onchain-receive.ts

```diff
@@ -152,7 +152,10 @@ it("identifies unconfirmed incoming on chain txn", async () => {
   expect(sendNotification.mock.calls[0][0].data.type).toBe("onchain_receipt_pending")
 
   const satsPrice = await getCurrentPrice()
-  const usd = (btc2sat(amount_BTC) * satsPrice!).toFixed(2)
+  if (!satsPrice) {
```

Comment:
Is this explicit enough? Does it only check for `null`/`undefined`?

Reply:
Well. JS is messed up, so this also check for 0, false, and NAN. The exception in this test actually used to ignore a falsy value in `satsPrice`. I think the test should fail in that case. No?